lib/components/edit-program-modal/index.tsx

```diff
@@ -0,0 +1,27 @@
+import ProgramForm from '../program-form';
```

Comment:
The edit-program-modal is exactly the same format as the create-program-modal. Is there a way we can abstract this?

Reply:
Imo it'd be better to keep it like this cuz the `EditProgramModal` is gonna eventually autofill its values and the `CreateProgramModal` isn't, and they're also gonna call different routes when the `Submit` button is clicked. If people think there's more abstracting that can be done though, I'm down to do it.